### PR TITLE
Add build steps

### PR DESCRIPTION
**Pull request by robdodson**

### CODE REVIEW

**Comment by robdodson** [comment]
@chuckh PTAL

File: README.md [hunk 1]
```diff
@@ -1,23 +1,78 @@
 # generator-polymer-init-custom-build
-> A starting point for building apps using polymer-build
 
-## Installation
+This template is a starting point for building apps using Polymer Starter Kit
+with a custom gulp process leveraging polymer-build, the library powering Polymer CLI
```
Comment (chuckh) [inline README.md]: Add period after Polymer CLI.

Comment (chuckh) [inline README.md]: Also add info about the additional functionality the gulpfile provides that the Polymer CLI does not, such as imagemin, clean, and clean before build.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,78 @@
...
+This command serves the minified version of the app generated using fragment bundling:
+
+    polymer serve build/bundled
+    # Open your browser and navigate to localhost:8080
+
+### Run tests
+
+This command will run
+[Web Component Tester](https://github.com/Polymer/web-component-tester) against the
+browsers currently installed on your machine.
```
Comment (chuckh) [inline README.md]: Should `machine.` be `machine:`. Check above use of periods and colons before code blocks.

**Comment by robdodson** [comment]
@chuckh updated, PTAL

**Comment by chuckh** [comment]
@robdodson nice work, lgtm.

**Pull request merged**